apps/AgeDetection/.gitignore

```diff
@@ -1,3 +1,5 @@
 # Ignore output binary
 age
 agedetection
+age_deploy.prototxt
+age_net.caffemodel
```

Comment:
It is always good to have a trailing newline at the end of a committed file (to avoid "false positives" regarding changed lines).

```suggestion
age_net.caffemodel

```

Reply:
Ok understood.